### PR TITLE
Update limitations

### PR DESCRIPTION
**Pull request by ajp-io**
Remove air gap limitation, because it's beta and so is the rest of EC. Update kURL migration limitation so it doesn't sound like it's impossible.

### CODE REVIEW

File: docs/vendor/embedded-overview.mdx [hunk 1]
```diff
@@ -43,13 +43,11 @@ As shown in the diagram above, the Embedded Cluster Config is included in the ap
 
 Embedded Cluster has the following limitations:
 
-* **No migration from kURL**: There is not a supported path to migrate an existing kURL instance to Embedded Cluster.
+* **No automated migration from kURL**: There is no automated migration from an existing kURL instance to Embedded Cluster. There is a relatively simple manual migration procedure for those interested.
```
Comment (paigecalvert) [inline docs/vendor/embedded-overview.mdx:46]: @ajp-io should we tell them to reach out to you for the steps?

Comment (ajp-io) [reply docs/vendor/embedded-overview.mdx:46]: Sure!

Comment (ajp-io) [reply docs/vendor/embedded-overview.mdx]: See what you think

**Review by paigecalvert — APPROVED** [review]
@ajp-io if you're cool with the edits to the manual migration procedure content, this is good to merge 👍

**Pull request merged**